ThomasShelby.java

```diff
@@ -13,16 +16,15 @@ private static void listTasks(Task[] taskManager, int taskCount) {
         }
     }
 
-    private static int addTask(Task[] taskManager, int taskCount, String[] cmdSplit) {
+    private static void addTask(String[] cmdSplit) {
         Task newTask = new Task(cmdSplit[1]);
         newTask.setDescription(cmdSplit[1]);
         taskManager[taskCount] = newTask;
         taskCount++;
         System.out.println("added: " + cmdSplit[1]);
-        return taskCount;
     }
 
-    private static void markTask(Task[] taskManager, String[] cmdSplit) {
+    private static void markTask(String[] cmdSplit) {
         int whichTask;
```

Comment:
The variable name whichTask sounds like a method that would return an integer. I suggest naming it taskNumber for enhanced readability